db/schema.rb

```diff
@@ -134,7 +134,7 @@
     t.index ["start_year"], name: "index_cohorts_on_start_year", unique: true
   end
 
-  create_table "cohorts_lead_providers", force: :cascade do |t|
+  create_table "cohorts_lead_providers", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
It probably doesn't affect anything but we shouldn't have this change I think?

Reply:
Oops, yes this shouldn't have been included.